### DISCUSSION TITLE
Improve layout extension doc

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#8448] [Feature Requests]
Please can you provide a detailed guide for extending tabular and/or card layout(s)? The actual example in the official doc barely scratches the surface of layout extending. A real-world example based on pre-installed layouts would be more useful.
Thanks!

**Comment by rijkvanzanten** [comment]
I have my own biases, but when you say "Extend", what do you mean exactly?

You are currently able to render an existing layout _within_ your layout, but that might not be the level of extending that you're looking for

**Reply by mhdaouas** [reply]
Adding action buttons for example or other UI elements like a chart.

**Reply by rijkvanzanten** [reply]
Interesting, yeah 👍🏻 

Currently, every layout is registered globally as `<layout-tabular />` (substitute the actual name), so you should be able to render any of the layouts nested as part of your own layout already (assuming you pass on all the props) to start adding stuff around it 🤔 

(Haven't really played with this too much myself yet 🙂 )

**Reply by mhdaouas** [reply]
Well the problem is that this is not documented at all. A good idea would be to add at least some examples of custom interfaces based on tabular/card layouts in the source code. I tried personally to recompile the tabular layout from the source code but failed miserably (I  think it's due to some missing dependencies because I get the error UNRESOLVED_IMPORT).

**Reply by rijkvanzanten** [reply]
Yes, because we've never even tried this ourselves haha!

**Reply by meerzulee** [reply]
> Well the problem is that this is not documented at all. A good idea would be to add at least some examples of custom interfaces based on tabular/card layouts in the source code. I tried personally to recompile the tabular layout from the source code but failed miserably (I think it's due to some missing dependencies because I get the error UNRESOLVED_IMPORT).

Did you able to reuse <layout-tabular/> component?

**Reply by mhdaouas** [reply]
I moved to something else and haven't tried yet, sorry. I will let you know if it works for me.

**Reply by divinepersuasionstudio** [reply]
> Interesting, yeah 👍🏻
> 
> Currently, every layout is registered globally as `<layout-tabular />` (substitute the actual name), so you should be able to render any of the layouts nested as part of your own layout already (assuming you pass on all the props) to start adding stuff around it 🤔
> 
> (Haven't really played with this too much myself yet 🙂 )

Please do tell, how do we properly use this globally registered layout? `<layout-tabular />`
May you provide an example <3

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
